db/seeds.rb

```diff
@@ -73,7 +74,19 @@
     available: true,
     condition: "mint",
     description: "Sakkusu Alto Saxophone. Solid construction and mechanism, comfortable key layout. Top F# key. This extra top note was introduced on most saxes in the 1960s and most teachers recommend it. High-Quality Pisoni Pads.",
-    image: Rails.root.join("app/assets/images/sakkusu_alto_saxophone.jpg").open,
+    image: "sakkusu_alto_saxophone.jpg",
```

Comment:
This is throwing a no file path error because this file path should be `sakkusu-alto-saxophone.jpg` 
I loved the addition of this seeding since Friday.